www/frontends/compiler_gym/src/components/ActionsNavbar.js

```diff
@@ -41,9 +41,10 @@ const CustomMenu = forwardRef(
   }
 );
 
+//name.toLowerCase().indexOf(this.props.filter.toLowerCase()) !== -1
+
```

Comment:
```suggestion
```

(I'm a little pedantic about committing commented-out code hehe)

Reply:
Noted and fixed